knowledge-portal/central/README.md

```diff
@@ -0,0 +1,17 @@
+# Setting up pipeline in central repository
+
+>You can just simply use the below code and paste it in your `.gitlab-ci.yml` file of your repo.
+
+```yaml
+stages:
+    - merge_request_to_source
```

Comment:
If above recommendation is being taken into consideration, you can also rename this stage to `push`